test/integration/testdata/egress-proxy.yaml.tmpl

```diff
@@ -8,7 +8,7 @@ spec:
   - name: http
     port: 80
   selector:
-    app: istio-egress
```

Comment:
lets just retain these names as is.. Even though its in istio namespace. k8s does the same.. kube-dns, kube-apiserver, etc.